.github/workflowsForDownstreamRepo/regression-test-downstream-repo.yaml

```diff
@@ -12,7 +12,7 @@ env:
   DEFAULT_UNDERLAYS: cmssynpuf,aouSR2019q4r4
   TANAGRA_UNDERLAY_FILES: cmssynpuf_verily,aouSR2019q4r4_verily
 jobs:
-  test:
+  regression-test:
```

Comment:
If you do change the job name here, please also change it downstream in `verily-src/verily-tanagra-backend` repo, so we can keep those two consistent.

Reply:
Done, https://github.com/verily-src/verily-tanagra-backend/pull/396